index.html

```diff
@@ -180,8 +180,10 @@ <h3>
         <p its-locale-filter-list="zh-hant" lang="zh-hant">原則上，中文排版所使用的漢字與標點符號比例皆為1:1的正方形，將其無縫隙並列排成版面。</p>
       </li>
     </ol>
-	      <div class="note translateme">
+	      <div class="note">
+          <p its-locale-filter-list="en" lang="en">For the most typographical rules described in this document, "regional differences" are more than "differences between Traditional Chinese". For example, although most publications in mainland China use simplified Chinese characters in horizontal writing mode, a few publications use traditional Chinese characters in horizontal writing mode or simplified characters in vertical writing mode. The typographical rules in mainland China, such as the punctuation position rules specified in <cite>General Rules for Punctuation</cite> (GB/T 15834—2011) also apply to vertical or traditional Chinese publications published in mainland China. For vertical and traditional Chinese publications published in Taiwan, Taiwan's typographic rules are used. Therefore, it is recommended that the user agents distinguish typographical rules by "region" rather than "traditional or simplified".</p>
```

Comment:
Thanks @r12a! I pushed a commit to fix the issues above.

-----

> This seems to imply that Mainland China uses TC in vertical writing. Is that correct? If so, better state it.

Yes. Do you mean that merely stating "a few publications use Traditional Chinese characters in horizontal writing mode or Simplified characters in vertical writing mode" is not enough? What else should I add? (This may be a separate issue.)

Reply:
If i'm correctly understanding the implications, i think i would change the text to read as follows (bolding used to mark the change):

> For example, although most publications in Mainland China use Simplified Chinese characters in horizontal writing mode **and Traditional characters in vertical mode**, a few publications use Traditional Chinese characters in horizontal writing mode or Simplified characters in vertical writing mode.

Reply:
Hmm, based on my understanding of the Chinese text (and my own experience), I think the situation in Mainland China should be like this:

![clreq](https://user-images.githubusercontent.com/2863444/75783728-faf69700-5d9b-11ea-99dc-a499e98c8d03.png)